yt_dlp/extractor/funker530.py

```diff
@@ -0,0 +1,75 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from .rumble import RumbleEmbedIE
+from .youtube import YoutubeIE
+from ..utils import clean_html
+
+
+class Funker530IE(InfoExtractor):
+    _VALID_URL = r'https?:\/\/(?:www\.)?funker530\.com\/video\/(?P<id>[^\/]+)\/?'
+    # TODO: `python test/test_download.py TestDownload.test_Funker530` only runs the first test?
```

Comment:
`python test/test_download.py TestDownload.test_Funker530_1` to run the 2nd test, and so forth